config/config.go

```diff
@@ -38,6 +38,8 @@ type Configuration struct {
 	DataCache       DataCache          `mapstructure:"datacache"`
 	StoredRequests  StoredRequests     `mapstructure:"stored_requests"`
 	CategoryMapping StoredRequestsSlim `mapstructure:"category_mapping"`
+	Accounts        StoredRequestsSlim `mapstructure:"accounts"`
```

Comment:
Ah... I assumed (I know, I know) it would be covered by `(StoredRequestsSlim*) validate()` but I am starting to think that might not actually exist. 
So this means probably CategoryMapping and VideoRequests aren't validated either.